website/project/metadata/osf-preregistration.json

```diff
@@ -0,0 +1,381 @@
+{
+    "name": "OSF Preregistration",
+    "version": 2,
+    "active": false,
+    "config": {
+	"hasFiles": true,
+        "fulfills": [{
+            "name": "OSF Preregistration",
+            "info": "https://cos.io/prereg/"
+        }]
+    },
+    "description": "OSF Preregistration (pre-data collection): You will be asked a series of questions to ensure that your sampling, design, and analysis plans are solidified prior to beginning your study.",
+    "pages": [{
+        "id": "page1",
+        "title": "Study Information",
+        "questions": [{
+            "qid": "q1",
+            "nav": "Title",
+            "type": "string",
+            "format": "text",
+            "title": "Title",
+            "description": "Provide the working title of your study. It is helpful if this is the same title that you submit for publication of your final manuscript, but it is not a requirement.",
+            "help": "Effect of sugar on brownie tastiness. <br /><br />How responses will be evaluated: The title should be a specific and informative description of a project. Vague titles such as 'Fruit fly preregistration plan' are not appropriate.",
+            "required": true
+        }, {
+            "qid": "q2",
+            "nav": "Authors",
+            "type": "string",
+            "title": "Authors",
+            "description": "The author who submits the preregistration is the recipient of the award money and must also be an author of the published manuscript. Additional authors may be added or removed at any time.",
```

Comment:
This needs to be changed since we can't mention "award money" in the OSF Preregistration form. I'll ask on the requirements doc what should be expected.